extensions/blocks/podcast-player/style.scss

```diff
@@ -70,16 +70,15 @@ $jetpack-podcast-player-error: $alert-red;
 		 * on links.
 		 */
 		a {
-			box-shadow: none;
-			text-decoration: none;
-			border: none;
-
+			&,
+			&:active,
+			&:visited,
```

Comment:
At this point, I'm wondering if things would be a bit cleaner if we moved this a level up and replaced all `&` with `a`. We gain nothing from the nested style, or do we?

Reply:
Updated in https://github.com/Automattic/jetpack/pull/15545/commits/406fbbb8796bc09125760c9c443c919c3a1f8538